src/com/trilead/ssh2/transport/MessageHandler.java

```diff
@@ -11,12 +11,4 @@
 public interface MessageHandler
 {
 	public void handleMessage(byte[] msg, int msglen) throws IOException;
-
-    /**
-     * Called to inform that no more messages will be delivered.
-     *
-     * @param cause
-     *      For diagnosis, the reason that caused the transport to close down.
-     */
-    public void handleEndMessage(Throwable cause) throws IOException;
```

Comment:
We can't remove a method from an interface: we'll break underlying implementations that depend on this method.